packages/fx-core/src/component/debugHandler/localEnvProvider.ts

```diff
@@ -0,0 +1,209 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+"use strict";
+
+import * as dotenv from "dotenv";
+import * as fs from "fs-extra";
+import * as os from "os";
+import * as path from "path";
+
+import { FolderName } from "../../common/local/constants";
+
+export interface LocalEnvs {
+  template: { [key: string]: string };
+  teamsfx: { [key: string]: string };
+  customized: { [key: string]: string };
+}
+
+export const LocalEnvKeys = Object.freeze({
+  frontend: {
+    template: {
+      Browser: "BROWSER",
+      Https: "HTTPS",
+      Port: "PORT",
+      SslCrtFile: "SSL_CRT_FILE",
+      SslKeyFile: "SSL_KEY_FILE",
+    },
+    teamsfx: {
+      ClientId: "REACT_APP_CLIENT_ID",
+      LoginUrl: "REACT_APP_START_LOGIN_PAGE_URL",
+      FuncEndpoint: "REACT_APP_FUNC_ENDPOINT",
+      FuncName: "REACT_APP_FUNC_NAME",
+    },
+  },
+  backend: {
```

Comment:
How to handle those place-holder envs like `SQL_xxx`?

Reply:
Should we put empty SQL envs in the file, or totally let user to manage the SQL envs?

Reply:
Better to have those when first time generating the file. We do let user to manage the values, but need place holders for keys.

Reply:
Added SQL env place holders when the project has SQL.